source/community/release_notes/redwood.rst

```diff
@@ -25,10 +66,140 @@ Instructor Experiences
+`contentstore.new_studio_mfe.use_new_course_outline_page <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L424>`_
+`contentstore.new_studio_mfe.use_new_course_team_page <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L464>`_
+`contentstore.new_studio_mfe.use_new_custom_pages <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L244>`_
+`contentstore.new_studio_mfe.use_new_export_page <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L364>`_
+`contentstore.new_studio_mfe.use_new_files_uploads_page <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L384>`_
+`contentstore.new_studio_mfe.use_new_grading_page <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L304>`_
+`contentstore.new_studio_mfe.use_new_group_configurations_page <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L524>`_
+`contentstore.new_studio_mfe.use_new_import_page <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L344>`_
+`contentstore.new_studio_mfe.use_new_schedule_details_page <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L264>`_
+`contentstore.new_studio_mfe.use_new_textbooks_page <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L504>`_
+`contentstore.new_studio_mfe.use_new_unit_page (disabled) <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L444>`_
+`contentstore.new_studio_mfe.use_new_updates_page <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L324>`_
+`contentstore.new_studio_mfe.use_new_video_uploads_page (disabled) <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L404>`_
+`new_studio_mfe.use_new_home_page <https://github.com/openedx/edx-platform/blob/f256684646aec6fd0d5519c6900ec99077e7db50/cms/djangoapps/contentstore/toggles.py#L225>`_
+
+Course authoring MFE config:
+
+ENABLE_UNIT_PAGE: "false"
+ENABLE_NEW_EDITOR_PAGES: "true"
+ENABLE_TAGGING_TAXONOMY_PAGES: "true"
+ENABLE_ASSETS_PAGE: "true"
+ENABLE_HOME_PAGE_COURSE_API_V2: "true"
```

Comment:
Can these be linked to their context, like the other waffles?

Reply:
I'm throwing out my current list in favor of what Maria posted.